data-prepper-plugins/common/src/test/java/com/amazon/dataprepper/plugins/sink/StdOutSinkTests.java

```diff
@@ -12,21 +13,47 @@
 package com.amazon.dataprepper.plugins.sink;
 
 import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.event.JacksonEvent;
 import com.amazon.dataprepper.model.record.Record;
 import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
You are using two different version of Junit now. I believe this is what is impacting your test coverage.

Reply:
Interesting

Reply:
Good catch! It's a subtle difference.

Reply:
> Thanks for making it support both! I have one comment on variable names and then I'll be ready to approve. It looks like the 90% coverage is working again, correct?

Yep .90 is working again